gatling-jms/src/main/scala/io/gatling/jms/request/JmsAttributes.scala

```diff
@@ -30,7 +30,7 @@ import io.gatling.core.session.Expression
 case class JmsAttributes(
   requestName:       String,
   destination:       JmsDestination,
-  replyDestination:  JmsDestination,
+  replyDestination:  Option[JmsDestination],
```

Comment:
Move `replyDestination` out of `JmsAttributes` so `RequestSend` doesn't have one and `RequestReply` has a mandatory one.